dispatch/static/manager/package.json

```diff
@@ -54,5 +55,8 @@
   },
   "bugs": {
     "url": "https://github.com/ubyssey/dispatch/issues"
+  },
+  "dependencies": {
+    "moment": "^2.17.1"
```

Comment:
is moment the only non dev dependency? why is the min version here different from on line 30 above?